### ISSUE TITLE
Issues concerning the online documentation on readthedocs

### ISSUE THREAD

**Issue by GitMensch** [LADSoft/OrangeC#99]
There seems to be something wrong in the GitHub setup (which I've originally provided but cannot change).

steps to reproduce:
* open https://orangec.readthedocs.io/en/latest/occ/OCC/
* open tools (click on bottom left)
* use a GitHub link, for example "View On GitHub"

Result:

Link to https://github.com/LADSoft/OrangeC/blob/master/home/docs/checkouts/readthedocs.org/user_builds/orangec/checkouts/latest/doc/occ/OCC.md

instead of the correct https://github.com/LADSoft/OrangeC/blob/master/doc/occ/OCC.md

Second issue: The PDF version of the documentation is currently not build.

**Comment by LADSoft** [comment]
I'm not sure how to resolve this.   To take the second issue first, the 'create PDF' box was clicked.   So I assumed the bad link was the problem.   However from the readthedocs documentation, they don't let you reconfigure this link.   They advise to delete the project and start over.   Which I did, but it didn't work any better than before

**Comment by LADSoft** [comment]
I did further research.   After reading this: https://github.com/rtfd/readthedocs.org/issues/1820 I came to the conclusion there are serious either in github or in readthedocs that prevent the link you reference from working.   Apparently, it goes in and out...  sometimes working and sometimes not.

I also found this in the readthedocs documentation:

> As MkDocs doesn’t support automatic PDF generation, Read the Docs cannot create a PDF version of your documentation with the Mkdocs option.

that jives with the fact that it just doesn't work even with the PDF selections set (which they are).

Both of these issues are beyond my control, and I'm closing this issue.
